### ISSUE TITLE
site gives user bad first code

### ISSUE THREAD

**Issue by vkmt** [airbrake/javabrake#62]
I signed up and started a javabrake project and got code that did not compile as the initial code from the airbrake website.
<img width="2243" alt="Screen Shot 2022-03-14 at 1 32 33 PM" src="https://user-images.githubusercontent.com/49798114/158258269-a9bc4c9d-f270-486b-bd5e-e59e348ebd16.png">

**Comment by scottsbaldwin** [comment]
The in-app instructions need to be updated to reflect what shows here: https://github.com/airbrake/javabrake#quickstart.

**Comment by thompiler** [comment]
Yes, I will update our instructions and post here once that's complete.

**Comment by thompiler** [comment]
>I verified that the site has been updated appropriately.

Thank you @vkmt!
